include/grpc/event_engine/event_engine.h

```diff
@@ -162,8 +162,11 @@ class EventEngine : public std::enable_shared_from_this<EventEngine> {
     /// Reads data from the Endpoint.
     ///
     /// When data is available on the connection, that data is moved into the
-    /// \a buffer, and the \a on_read callback is called. The caller must ensure
-    /// that the callback has access to the buffer when executed later.
+    /// \a buffer, and the \a on_read callback is called. If the read succeeds
```

Comment:
"and the \a on_read callback is called" should probably be removed

Reply:
Removed.